### ISSUE TITLE
What if the new classes all are out of pretrain domain?

### ISSUE THREAD

**Issue by sipie800** [zhoudw-zdw/RevisitingCIL#4]
Hello! I'm doing some landing work, which could benefit from CIL. I've read your paper. The method is quite simple and impressive.

However there is one thing I'm interested in. In your work, the new classes do seem to be close to the PTM domain. That makes your method seems to be something like a transfer learning. If new classes are all far from the pretrain classes (far from any classes in ImageNet,Cifar). Can the method deal with domain gap ? If not, can the incremental classes have the nice metric in paper?

**Comment by zhoudw-zdw** [comment]
Hi, thanks for your interest in our work.

In fact, we have tried many datasets that are out of the training distribution, e.g., ImageNet-R/A and ObjectNet. ADAM works competitively in these scenarios.

If you are working on a project in the real world, I suggest using stronger backbones like CLIP pre-trained encoder and combining them with ADAM.

**Comment by sipie800** [comment]
> Hi, thanks for your interest in our work.
> 
> In fact, we have tried many datasets that are out of the training distribution, e.g., ImageNet-R/A and ObjectNet. ADAM works competitively in these scenarios.
> 
> If you are working on a project in the real world, I suggest using stronger backbones like CLIP pre-trained encoder and combining them with ADAM.

Thank you for reply.
Maybe I didn't clarify my point. IMO actually an ObjectNet or ImageNet class is in the same domain with a ImageNet or COCO etc. 
I mean a true domain gap in real world. Say a dataset of industrial or medical or other rare seen objects in daily life.
One couldn't count on a ImageNet PTM to deal with so many domain in real world. Neither a CLIP one or any PTM which is feed with samples collected from internet. The internet itself is a narrow domain!
IL is the very fundemental method in future. Industry really is looking forward to these method. So please go further from the benchmark things if you guy are willing to.
Anyway I'm gonna try out your method on these domain.
